dmc_configs/configs_fig_7.py

```diff
@@ -0,0 +1,68 @@
+"""
+Implements `dist_agent_1lm_randrot_nohyp_x_percent_20p`
+"""
+
+import copy
+from dmc_configs.dmc_eval_experiments import dist_agent_1lm_nohyp_randrot
+
+
+def update_x_percent_threshold_in_config(
+    config, x_percent_threshold, evidence_update_threshold="x_percent_threshold"
+):
+    """Update the x_percent threshold in the config.
+
+    Args:
+        config (dict): The config to update.
+        x_percent_threshold (float): The percentage of the threshold to update.
+
+    Returns:
+        dict: The updated config.
+    """
+    # Update the run name
+    config[
+        "logging_config"
+    ].run_name = f"dist_agent_1lm_randrot_nohyp_x_percent_{x_percent_threshold}p"
+
+    # Update the x_percent_threshold
+    lm_config_dict = config["monty_config"].learning_module_configs
+    lm_config_dict["learning_module_0"]["learning_module_args"][
+        "evidence_update_threshold"
+    ] = x_percent_threshold
+
+    # Update the string value for evidence_update_threshold
+    lm_config_dict["learning_module_0"]["learning_module_args"][
+        "evidence_update_threshold"
+    ] = evidence_update_threshold
+    return config
+
+
+dist_agent_1lm_randrot_nohyp_x_percent_5p = copy.deepcopy(dist_agent_1lm_nohyp_randrot)
+dist_agent_1lm_randrot_nohyp_x_percent_5p = update_x_percent_threshold_in_config(
+    dist_agent_1lm_randrot_nohyp_x_percent_5p, 0.05, "x_percent_threshold"
```

Comment:
Note that `x_percent_threshold` should be passed in as an actual percentage (e.g. 5, or 20, not 0.05 or 0.20), hence why typical values are 10 or 20, see also how it is used in `EvidenceGraphLM`.